python/cudf/benchmarks/API/bench_functions.py

```diff
@@ -9,7 +9,9 @@
 from utils import benchmark_with_object
 
 
-@pytest_cases.parametrize_with_cases("objs", prefix="concat")
+@pytest_cases.parametrize_with_cases(
+    "objs", prefix="concat", cases="bench_functions"
```

Comment:
I expected this to match `cases_dataframe` above. Is this right?
```suggestion
    "objs", prefix="concat", cases="cases_functions"
```

Reply:
ugh yes you're totally right. I must have accidentally undone this reverting another change. Thank you.